src/bentoml/_internal/frameworks/transformers.py

```diff
@@ -34,50 +79,53 @@
         "'transformers' is required in order to use module 'bentoml.transformers'. Install transformers with 'pip install transformers'."
     )
 
+if t.TYPE_CHECKING:
+    TransformersPreTrained = (
+        transformers.PreTrainedTokenizerBase
+        | transformers.PreTrainedTokenizer
+        | transformers.PreTrainedTokenizerFast
+        | transformers.PreTrainedModel
+        | transformers.TFPreTrainedModel
+        | transformers.FlaxPreTrainedModel
+        | transformers.image_processing_utils.BaseImageProcessor
+        | transformers.SequenceFeatureExtractor
+        | transformers.Pipeline
+    )
+
 
 __all__ = ["load_model", "save_model", "get_runnable", "get"]
 
+_object_setattr = object.__setattr__
 
 MODULE_NAME = "bentoml.transformers"
-API_VERSION = "v1"
+API_VERSION = "v2"
```

Comment:
Will we have backward compatibility with `v1`?

Reply:
Yes. It is already included.